net-p2p/automatic/automatic-9999.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+EGIT_REPO_URI="https://github.com/1100101/${PN^}.git"
```

Comment:
If you must add a live ebuild, please use a live template (if [[ ${PV} == .. ]]) to make it easier to maintain.

Reply:
Live ebuild removed.